test/taskcreator_test.js

```diff
@@ -207,6 +208,21 @@ suite('taskcreator_test.js', function() {
           (err) => { debug('Got expected error: %s', err); });
       });
     }
+
+    test('adds a new row to lastFire', async function() {
```

Comment:
Let's do *two* appends for different hookIds, and see if they can be fetched independently.

Reply:
keeping this test and adding a new test as well.